app/commands/mentor/testimonial/create.rb

```diff
@@ -0,0 +1,15 @@
+class Mentor::Testimonial::Create
+  include Mandate
+
+  initialize_with :discussion, :testimonial
+
+  def call
+    Mentor::Testimonial.create!(
+      mentor: discussion.mentor,
+      student: discussion.student,
+      discussion:,
+      content: testimonial
+    )
+    User::ResetCache.(discussion.mentor)
```

Comment:
I've not made this into a job yet. Do we want that? I think the queries are quite fast.

Reply:
Yeah, job. But you can just call `defer` instead of `.()` here.